examples/simple-udp/main.go

```diff
@@ -176,6 +176,52 @@ func readWriteLoop(conn net.PacketConn, stop chan struct{}, s *sdk.SDK) {
 				respond(conn, sender, "ERROR: Invalid ANNOTATION command, must use zero or 2 arguments\n")
 				continue
 			}
+		case "PLAYER_CAPACITY":
+			switch len(parts) {
+			case 1:
+				respond(conn, sender, getPlayerCapacity(s))
+				continue
+			case 2:
+				if cap, err := strconv.Atoi(parts[1]); err != nil {
+					respond(conn, sender, err.Error()+"\n")
+					continue
+				} else {
+					setPlayerCapacity(s, int64(cap))
+				}
+			default:
+				respond(conn, sender, "ERROR: Invalid PLAYER_CAPACITY, should have 0 or 1 arguments\n")
+				continue
+			}
+
+		case "PLAYER_CONNECT":
+			if len(parts) < 2 {
+				respond(conn, sender, "ERROR: Invalid PLAYER_CONNECT, should have 1 arguments\n")
+				continue
+			}
+			playerConnect(s, parts[1])
+
+		case "PLAYER_DISCONNECT":
+			if len(parts) < 2 {
+				respond(conn, sender, "ERROR: Invalid PLAYER_CONNECT, should have 1 arguments\n")
+				continue
+			}
+			playerDisconnect(s, parts[1])
+
+		case "PLAYER_CONNECTED":
+			if len(parts) < 2 {
+				respond(conn, sender, "ERROR: Invalid PLAYER_CONNECTED, should have 1 arguments\n")
+				continue
+			}
+			respond(conn, sender, playerIsConnected(s, parts[1]))
+			continue
+
+		case "GET_PLAYERS":
+			respond(conn, sender, getConnectedPlayers(s))
```

Comment:
There should be new line symbol to comply with other responses and for better output in `nc -u`. Something like:
```suggestion
			respond(conn, sender, fmt.Sprintf("%s\n", getConnectedPlayers(s)))
```